File: Day9/Solution.cpp
#include <iostream>

using namespace std;

int factorial(int n){
    int fact = 1;
    if (n == 1){
        return 1;
    }else{
        for(int i = 1; i<=n; ++i){
            fact = fact*i;
        }
        return fact;
    }
}

int main(){
    int N;
    cin >> N;

    cout<<factorial(N);

    return 0;
}

File: Day9/Solution.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();

        in.close();
        System.out.println(factorial(n));
    }

    private static int factorial(int n){
        if ( n == 1){
            return 1;
        } else {
            int i, fact=1;
            for(i = 1; i<=n; i++){
                fact = fact*i;
            }

            return fact;
        }
    }
}